src/constants.ts

```diff
@@ -0,0 +1,11 @@
+export const OUTPUT_TEMPLATE_NAME = 'daily_output.base.json';
+export const OUTPUT_NAME = 'daily_output.json';
+export const ONE_GiB = 134_217_728;
+export const GROUP_EFFORT_REWARDS = [
+	75, 55, 50, 47, 45, 43, 40, 37, 35, 32, 31, 30, 29, 28, 27, 26, 25, 24, 23, 22, 21, 20, 19, 18, 17, 16, 15, 14, 13,
+	12, 11, 10, 9, 9, 8, 8, 7, 7, 6, 6, 5, 5, 4, 4, 3, 3, 2, 2, 1, 1
+] as const;
```

Comment:
Let's either generate this array algorithmically or carefully review all of the 50 items.

Reply:
Just copied/pasted the table values here and found out that the values are OK. Verified.